auth-web/src/views/auth/MaintainBusinessView.vue

```diff
@@ -36,6 +36,50 @@
           <learn-more-button
            :redirect-url="learnMoreUrl"
           />
+          <p v-if="!userProfile" class="mt-5">Don't have your Cooperative Passcode?
+            <v-menu top offset-y v-model="contactUsPopover" :close-on-content-click="false" attach="#maintain-info-container">
+              <template v-slot:activator="{ on, attrs }">
+                <span v-bind="attrs" v-on="on" class="popover-activator">Contact Us</span>
+              </template>
+              <v-card class="contact-popover">
+                <v-list class="pa-5" max-width="30rem">
+                  <v-list-item class="pb-3">
+                    <v-list-item-title class="popover-title mt-5 font-weight-bold">Don't have a Passcode for your Cooperative<br>Association?</v-list-item-title>
+                    <v-list-item-action class="mt-0">
+                      <v-icon @click="contactUs = false" color="bcgovblueLink">mdi-close</v-icon>
+                    </v-list-item-action>
+                  </v-list-item>
+                  <v-list-item>
+                    <v-list-item-content>
+                      <v-list-item-subtitle>If you have not recieved your Access Letter from BC Registries,<br>or have lost your Passcode, please contact us at:</v-list-item-subtitle>
+                    </v-list-item-content>
+                  </v-list-item>
+                  <v-list-item>
+                    <v-list-item-content>
+                      <v-list-item-subtitle>Toll Free: 1-877-526-1526</v-list-item-subtitle>
+                      <v-list-item-subtitle class="my-1">Phone: 250 387-7848</v-list-item-subtitle>
```

Comment:
You can use "tel:" hrefs to make a callable link (nice on mobile). You can see how the Entity team did it in their UIs.

Reply:
oo ok, will do that!

Reply:
Updated